src/sinks/gcp/mod.rs

```diff
@@ -90,11 +90,19 @@ fn get_token_implicit() -> Result<Token, GcpError> {
     }
 }
 
+fn get_token_with_creds_blocking(
+    jwt: &Jwt<JwtClaims>,
+    credentials: &Credentials,
+) -> Result<Token, GoErr> {
+    let mut rt = tokio_compat::runtime::current_thread::Runtime::new()?;
```

Comment:
I'm certainly not excited about this, but I understand it's necessary and I see the elasticsearch sink has to do the same thing..

Reply:
We should remove this once sink build function will be async. Here also `Credentials::from_file` and `get_token_implicit` which can be async.

Reply:
Exactly right.